src/components/BorrowerProfile/LendCta.vue

```diff
@@ -69,19 +70,24 @@
 								${{ priceOption }}
 							</option>
 						</kv-ui-select>
+
 						<!-- Lend button -->
+						<!-- eslint-disable max-len -->
 						<kv-ui-button
-							v-if="!isAdding && this.amountInBasket === '' && !this.lentPreviously"
+							v-if="!isAdding && this.amountInBasket === '' && !this.lentPreviously && this.status !== 'funded' && this.status !== 'refunded' && this.status !== 'expired' && !this.allSharesReserved"
```

Comment:
Can we create a couple computed properties to help with these multi-state checks for visibility? The inverse of this is reused below so a single computed property could power both cases.